src/chapters/git/branches.rst

```diff
@@ -0,0 +1,248 @@
+      of the output.
+
+OK. We created a new branch, added a file, put in some code, and committed all
+of those changes to the repository. What's going on back in ``main``?
+
+Switching Between Branches
+--------------------------
+
+Now that ``git_practice`` has two different branches, we can switch between
+them with the syntax:
+
+.. sourcecode:: bash
+
+   git checkout branch-name
+
+Enter ``git checkout main`` into the terminal. Pay close attention to the
+file tree in VS Code and the ``greeting.py`` tab in the editor.
+
+.. figure:: figures/git-checkout.gif
+   :alt: Switching between branches updates the files and code in the project.
+   :width: 70%
+
+   Branches contain different versions of the project files and code.
+
+Notice that ``greeting.py`` disappears from the file tree. Also, note that
+``(deleted)`` appears in its editor tab.
+
+Fear not! Our work hasn't gone away. Enter ``git checkout hello-branch`` in the
+terminal to see the file and its code reappear.
+
+The ``checkout`` command switches between branches. When we first made
+``hello-branch``, it was identical to ``main``. However, once we commit some
+changes, the two branches no longer match. By flipping back and forth between
+them, we call up different versions of our project.
+
+.. admonition:: Try It!
+
+   #. Return to the ``main`` branch in the ``git_practice`` repository.
+   #. Create a new branch off of ``main``.
+   #. In the new branch, add some comments to ``num_guess.py``.
+   #. Save and commit the changes.
+   #. Use ``git branch`` to see the list of branches in the repo.
+   #. Use ``git checkout`` to switch between the branches. Notice how the file
+      tree and code both update.
+
+Check Your Understanding
+------------------------
+
+.. admonition:: Question
+
+   What is a reason for creating a branch in Git?
+
+   .. raw:: html
+
+      <ol type="a">
+         <li><input type="radio" name="Q1" autocomplete="off" onclick="evaluateMC(name, false)"> To split work between team members.</li>
+         <li><input type="radio" name="Q1" autocomplete="off" onclick="evaluateMC(name, false)"> To try out a new feature before adding it to the main program.</li>
+         <li><input type="radio" name="Q1" autocomplete="off" onclick="evaluateMC(name, false)"> To avoid conflicts in the repo history.</li>
+         <li><input type="radio" name="Q1" autocomplete="off" onclick="evaluateMC(name, true)"> All of the above.</li>
+      </ol>
+      <p id="Q1"></p>
+
+.. Answer = d
+
+.. admonition:: Question
+
+   Do you think we can create a branch off of a branch, like this?
+
+   .. figure:: figures/branches-cc.png
+      :alt: A two-level branch: Branch 1 splits off of main, and brach 2 splits off of branch 1.
+      :width: 60%
+
+   .. raw:: html
+
+      <ol type="a">
+         <li><input type="radio" name="Q2" autocomplete="off" onclick="evaluateMC(name, true)"> Yes we can, but we should really keep branches only one layer deep.</li>
```

Comment:
I think I recall you telling me not to branch off of a branch...  This happened after I complicated the PR process by doing the two-level branch instead of splitting off of main.  ;-)

Anyway, my thinking lines up with yours about the novice Git users.  They'll learn different practices when they need to, but for now, simple is better.  Sometimes, 100% accuracy takes a hit in favor of simplicity for beginners.